src/ws-header/storage/cookie-storage.js

```diff
@@ -0,0 +1,66 @@
+import {AbstractStorage} from './abstract-storage';
```

Comment:
This classes are to abstract the platform access to localStorage and Cookies. This makes code cleaner, better testable and more self explanatory